composer.json

```diff
@@ -23,7 +23,8 @@
         "symfony/finder": "^2.2 || ^3.0",
         "symfony/polyfill-php54": "^1.0",
         "symfony/process": "^2.3 || ^3.0",
-        "symfony/stopwatch": "^2.5 || ^3.0"
+        "symfony/stopwatch": "^2.5 || ^3.0",
+        "symfony/polyfill-php55": "^1.3"
```

Comment:
how did you add it?
why https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/composer.json#L37 didn' t work ?